src/index.html

```diff
@@ -3,11 +3,46 @@
   <head>
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <title>Product cards</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,400;0,500;0,700;1,400;1,500;1,700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./styles/index.scss">
   </head>
   <body>
-    <h1>Product cards</h1>
+    <main class="main">
+      <card class="card" data-qa="card">
+        <img class="card__image" src="./images/imac.jpeg"
+         alt="iMac 27">
+
+        <div class="card__title">
+          <h1 class="card__title-title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h1>
+        </div>
+
+        <div class="card__product-code">Product code: 195434</div>
+
+        <div class="card__reviews">
+          <div class="card__reviews-stars stars">
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+            <div class="stars__star"></div>
+          </div>
+
+          <p class="card__reviews-counter">Reviews: 5</p>
+        </div>
+
+        <div class="card__price">
+          <p class="card__price-text">Price:</p>
+          <p class="card__price-sum">$2,199</p>
```

Comment:
```suggestion
          <p class="card__price-amount">$2,199</p>
```
I think it's better like that 👍